sddi-base/Dockerfile

```diff
@@ -191,7 +191,7 @@ RUN set -ex && \
   ckan config-tool "${CKAN_INI}" "ckanext.spatial.search_backend = solr-bbox" && \
   ckan config-tool "${CKAN_INI}" "scheming.dataset_schemas = ckanext.scheming:ckan_dataset.yaml" && \
   ckan config-tool "${CKAN_INI}" "scheming.presets = ckanext.scheming:presets.json ckanext.repeating:presets.json ckanext.composite:presets.json" && \
-  ckan config-tool "${CKAN_INI}" "scheming.dataset_fallback = false" && \
+  ckan config-tool "${CKAN_INI}" "scheming.dataset_fallback = false" && \ 
```

Comment:
Remove sapce at the end of file.
Check your VS-code settings, this can be automated. This is important, to keeps diffs between versions as clean as possible.

Reply:
It is removed